smithy-aws-traits/src/main/resources/META-INF/smithy/aws.protocols.smithy

```diff
@@ -123,7 +123,42 @@ structure ec2Query {}
 string ec2QueryName
 
 /// Indicates that an operation supports checksum validation.
-@trait(selector: "operation")
+@trait(
+    selector: "operation",
+    breakingChanges: [
+        {
+            change: "remove",
+            severity: "ERROR",
```

Comment:
Should we include a message when the entire trait is removed?

Reply:
oh I wanted to add message later after adding test, but forgot. thanks for catching.